middlewares/auth.js

```diff
@@ -21,4 +21,12 @@ const authMiddleware = (req, _res, next) => {
   next()
 }
 
-module.exports = authMiddleware
+const restrictTo = (...roles) => {
```

Comment:
Better move it to another file and rename to permissionMiddleware

Reply:
you think so? it's part of auth middleware after all, just extends it